### DISCUSSION TITLE
Constant is define to 1h in api

### DISCUSSION THREAD

**Discussion by damienfernandes** [Boavizta/cloud-scanner#328] [Q&A] [answered]
Hello,

Why the constant is define to 1h in api ? see code [here](https://github.com/Boavizta/cloud-scanner/blob/9dfff1d90a426dc4ab8390bf51738adfb3b4afe7/cloud-scanner-cli/src/metric_server.rs#L49)
I would like to define metrics values to 1 min.

Thanks in advance

**Comment by demeringo** [comment, marked as answer]
The rationale behind this hardcoded value was that I initialy only expected to produce metrics for one hour of use (and that it eased the question of units when displaying results in a dashboard ;-) 

You are right, this could accept a parameter (like in CLI mode).

**Comment by demeringo** [comment]
Change merged in #338
